Competition/src/main/java/competition/subsystems/elevator/commands/ControlElevatorWithGamepadCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.elevator.commands;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.elevator.ElevatorSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ControlElevatorWithGamepadCommand extends BaseCommand {
+    final OperatorInterface oi;
+    final ElevatorSubsystem elevatorSubsystem;
+
+    public ControlElevatorWithGamepadCommand(OperatorInterface oi, ElevatorSubsystem elevatorSubsystem) {
+        this.oi = oi;
+        this.elevatorSubsystem = elevatorSubsystem;
+        this.requires(elevatorSubsystem);
+
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+
```

Comment:
Here we should read from a joystick value, and call setPower on the elevator. I think we're going to need to add a second gamepad now - we are getting a lot of commands.

Reply:
I removed this command, can reimplement if needed